### ISSUE TITLE
Outputs are wrong during model.eval()

### ISSUE THREAD

**Issue by pfunami** [vqdang/hover_net#245]
Hello. Thank you for your great work!

I have tried inference with `hovernet_fast_monusac_type_tf2pytorch.tar` by the original code.
However, the predicted instance map was empty.
After some trial and error, I realized that the output changed depending on whether or not `model.eval()` was declared.

First, the below right image is a sample predicted instance map without `model.eval()` .
![image](https://user-images.githubusercontent.com/40795078/222692225-79faeb78-dc10-4775-afa3-350d25af9d51.png)

Next, the below image is with `model.eval()`.
![image](https://user-images.githubusercontent.com/40795078/222692700-4a92711f-8723-4bbf-a969-45db9b72af44.png)

I think the difference in HoVerNet with and without `model.eval()` is in batch normalization.
This is because batch normalization behaves differently during training and inference.

So I looked at the code related to inference in this repository to see if anything was done to handle batch normalization during inference.
However, I did not see any such processing...

Is there any knowledge to make it work well even if `model.eval()` is specified?
Or is the only way to make it reason without specifying `model.eval()` (i.e. in training mode)?

My environment is as follows
Python 3.8.12
PyTorch 1.11.0

Also, my implementation is as follows.
```
model = HoVerNet(mode='fast', nr_types=nr_types)
model.load_state_dict(torch.load(chkpt_path)["desc"])
model = model.to(device)
model = DataParallel(model)
model.eval() # or not

image_dataset = ImageToPatchDataset(input_image)
image_loader = DataLoader(
    image_dataset,
    shuffle=False,
    batch_size=batch_size,
    collate_fn=collate,
    pin_memory=True
)

pred_map = torch.empty(
    size=(image_dataset.max_x_coord, image_dataset.max_y_coord, 3),
    dtype=torch.float32,
    device=device
)

for coords, image_batch in tqdm(image_loader):
    image_batch = image_batch.to(device)
    with torch.no_grad():
        pred_dict = model(image_batch)
        pred_dict = OrderedDict(
            [[k, v.permute(0, 2, 3, 1).contiguous()] for k, v in pred_dict.items()]
        )

        np_pred = F.softmax(pred_dict['np'], dim=-1)[..., 1:]
        hv_pred = pred_dict['hv']
        for i in range(image_batch.shape[0]):
            left = coords[i][0]  # left, bottom, right, top
            bottom = coords[i][1]
            right = coords[i][2]
            top = coords[i][3]
            pred_map[bottom:top, left:right, 0] = np_pred[i, :, :, 0]
            pred_map[bottom:top, left:right, 1] = hv_pred[i, :, :, 0]
            pred_map[bottom:top, left:right, 2] = hv_pred[i, :, :, 1]

pred_map = pred_map.cpu().detach().numpy()
pred_map = pred_map[: image_dataset.im_h, : image_dataset.im_w, :]
instance_map = process_instance(pred_map) # post process instance map
```

**Comment by pfunami** [comment]
I closed this because I thought the execution method was different from what the repository intended.
